_posts/13-03-01-Docker.md

```diff
@@ -5,41 +5,36 @@ anchor:  docker
 
 ## Docker {#docker_title}
 
-Beside using Vagrant, another easy way to get a virtual development or production environment up and running is [Docker].
-Docker helps you to provide Linux containers for all kind of applications.
-There are many helpful docker images which could provide you with other great services without the need to install
-these services on your local machine, e.g. MySQL or PostgreSQL and a lot more. Have a look at the [Docker Hub Registry]
-[docker-hub] to search a list of available pre-built containers, which you can then run and use in very few steps.
+[Docker] is so called because it's all about containers. It's a lightweight alternative to a fully-fledged Vagrant VM. PHP's OOP is perhaps a helpful analogy to understanding it: think of Docker "images" as classes and "containers" as instances of them. For a typical LAMP application, you might have a web server container, another for the PHP-FPM process and a third for the MySQL server. Your project files - PHP, HTML, other assets, database contents – all remain on your host computer. You can create containers from the command line, or build a `docker-compose.yml` file that specifies which to use and how they communicate ("link") with one another.
```

Comment:
In general I think this paragraph is rather complex. I can have a go at shuffling it around when im not super jetlagged, but I do like where you are heading.